app/services/proofing/aamva/proofer.rb

```diff
@@ -50,8 +50,7 @@ def initialize(config)
       end
 
       def proof(applicant)
-        aamva_applicant =
-          Aamva::Applicant.from_proofer_applicant(OpenStruct.new(applicant))
```

Comment:
This is the scariest removal, but in looking through the usage, it doesn't seem like we call methods on it at all, and only use hash key access.

https://github.com/18F/identity-idp/blob/ca1ce25/app/services/proofing/aamva/applicant.rb#L30-L43